### PR TITLE
test_setup:Introducing a new param 'vt_ulimit_memlock'

### PR DESCRIPTION
**Pull request by yanglei-rh**
Introducing a new param `vt_ulimit_memlock`.

ID:2133389
Signed-off-by: Lei Yang <leiyang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Hello @luckyh @PaulYuuu Could you please help me review this patch when you have free time? Many thanks.

**Review by PaulYuuu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
Oh, I think you need to add the description in the class documentation.

**Comment by yanglei-rh** [comment]
> Oh, I think you need to add the description in the class documentation.

Thanks for your reminder. Updated.

**Comment by yanglei-rh** [comment]
Hello @YongxueHong @luckyh Please help me review it again, thanks a lot.

File: virttest/test_setup.py [hunk 1]
```diff
@@ -2559,10 +2559,12 @@ class UlimitConfig(Setuper):
                     that the current process can create.
     vt_ulimit_nofile: The maximum number of open file descriptors
                       for the current process.
+    vt_ulimit_memlock: The maximum size a process may lock into memory
```
Comment (luckyh) [inline virttest/test_setup.py:2562]: Missing the trailing `.`

Comment (yanglei-rh) [reply virttest/test_setup.py]: Thanks for your reminder, updated.

**Review by luckyh — APPROVED** [review]

**Comment by YongxueHong** [comment]
Hi @PaulYuuu 
Any comment? Thanks.

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Comment by PaulYuuu** [comment]
> Hi @PaulYuuu Any comment? Thanks.

No, thanks.

**Pull request merged**